packages/image-block/src/components/ResponsiveImage.spec.ct.tsx

```diff
@@ -0,0 +1,108 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Would be nice to have these new tests written with vitest

Reply:
Added the tests in vitest